diff --git a/src/client_list.c b/src/client_list.c
index 015dd95..148a590 100644
--- a/src/client_list.c
+++ b/src/client_list.c
@@ -33,7 +33,6 @@
 #include <pthread.h>
 #include <sys/wait.h>
 #include <sys/types.h>
-#include <sys/unistd.h>
 
 #include <string.h>
 
diff --git a/src/firewall.c b/src/firewall.c
index 9c53d02..c6592de 100644
--- a/src/firewall.c
+++ b/src/firewall.c
@@ -35,7 +35,6 @@
 #include <pthread.h>
 #include <sys/wait.h>
 #include <sys/types.h>
-#include <sys/unistd.h>
 
 #include <string.h>
 
diff --git a/src/util.c b/src/util.c
index 506c706..a4d2e76 100644
--- a/src/util.c
+++ b/src/util.c
@@ -36,7 +36,6 @@
 #include <sys/wait.h>
 #include <sys/types.h>
 #include <sys/time.h>
-#include <sys/unistd.h>
 #include <netinet/in.h>
 #include <sys/ioctl.h>
 #include <arpa/inet.h>
