diff --git a/src/client_list.c b/src/client_list.c
index c0e3e2b..267155c 100644
--- a/src/client_list.c
+++ b/src/client_list.c
@@ -33,7 +33,7 @@
 #include <pthread.h>
 #include <sys/wait.h>
 #include <sys/types.h>
-#include <sys/unistd.h>
+#include <unistd.h>
 
 #include <string.h>
 
diff --git a/src/debug.c b/src/debug.c
index ba84d02..9a690e6 100644
--- a/src/debug.c
+++ b/src/debug.c
@@ -31,6 +31,7 @@
 #include <time.h>
 #include <unistd.h>
 #include <signal.h>
+#include <pthread.h>
 
 #include "conf.h"
 
diff --git a/src/firewall.c b/src/firewall.c
index 9c53d02..c6592de 100644
--- a/src/firewall.c
+++ b/src/firewall.c
@@ -35,7 +35,6 @@
 #include <pthread.h>
 #include <sys/wait.h>
 #include <sys/types.h>
-#include <sys/unistd.h>
 
 #include <string.h>
 
diff --git a/src/util.c b/src/util.c
index b604227..4e85fbf 100644
--- a/src/util.c
+++ b/src/util.c
@@ -38,7 +38,7 @@
 #include <pthread.h>
 #include <sys/wait.h>
 #include <sys/types.h>
-#include <sys/unistd.h>
+#include <unistd.h>
 #include <netinet/in.h>
 #include <sys/ioctl.h>
 #include <arpa/inet.h>
